.babelrc

```diff
@@ -8,6 +8,7 @@
         }
       }
     ],
+    "@babel/preset-react",
```

Comment:
Required to enable tests that use JSX, otherwise we'll get complaints from Jest about unrecognized characters (like `<`).